config.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+/* eslint no-process-env: 0 */
+
+const env = process.env.NODE_ENV || 'production';
+
+module.exports = {
+  dateLocales: 'en-GB',
+  dateFormat: {
+    day: '2-digit',
+    month: 'long',
+    year: 'numeric'
+  },
+  env: env,
+  govukNotify: {
+    notifyApiKey: process.env.NOTIFY_KEY,
+    caseworkerEmail: process.env.CASEWORKER_EMAIL,
+    userConfirmationTemplateId: process.env.USER_CONFIRMATION_TEMPLATE_ID,
+    businessConfirmationTemplateId: process.env.BUSINESS_CONFIRMATION_TEMPLATE_ID
+  },
+  hosts: {
+    acceptanceTests: process.env.ACCEPTANCE_HOST_NAME || `http://localhost:${process.env.PORT || 8080}`
+  },
+  redis: {
+    port: process.env.REDIS_PORT || '6379',
+    host: process.env.REDIS_HOST || '127.0.0.1'
+  },
+  upload: {
+    maxFileSizeInBytes: 20 * 1024 * 1024, // 20MiB in bytes
+    hostname: process.env.FILE_VAULT_URL,
+    allowedMimeTypes: [
+      'image/jpeg',
+      'image/jpg',
+      'image/png',
+      'image/gif'
+    ],
+    documentCategories: {
+      'identity-documents': {
+        limit: 6,
+        limitValidationError: 'maxIdDocsUploads'
+      }
+    }
```

Comment:
is this config relevant to uploads in this form?

Reply:
the requirements are not clear if there should be a limit on uploaded files in this category, I'll confirm this as a part of [FMRF-18](https://collaboration.homeoffice.gov.uk/jira/browse/FMRF-18) 
but can remove it for now to avoid confusion.

Reply:
Confirmed that the maximum limit is one file, this will remain and be covered in a separate PR.